.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
+<describe the changes you have made here>
```

Comment:
Describe the changes is enough, maybe just add a simple:
Why did you make the changes?  -> The one could add here the issue ref or just describe a one liner.
I think motivation, considered alternatives are only usefuil for major architecture changes